ip/ip.go

```diff
@@ -17,26 +15,11 @@ type addr struct {
 
 func GetIpAddress(vmName string) (string, error) {
 
-	health, err := hypervisor.HealthCheck(vmName)
+	err := hypervisor.HealthCheck(vmName)
```

Comment:
status should also be checked here

```
status, err := hypervisor.HealthCheck(vmName)
if err != nil {
    return "", err
}

if !status {
    return "", fmt.Errorf("VM health check failed")
}
```

Reply:
everything else looks good!

Reply:
hypervisor.HealthCheck utility function only returns as error. 

so the HealthCheck method of the type Hypervisor return (bool, error)
and the HealthCheck function returns error only.

Reply:
ahh right cool, merging this then